src/control-plane/services.ts

```diff
@@ -21,6 +21,7 @@ export class Services extends Construct {
 
   constructor(scope: Construct, id: string, props: ServicesProps) {
     super(scope, id);
+    addTemplateTag(this, 'Services');
```

Comment:
Same comment as for the ControlPlaneAPI tag.

Reply:
Removed